index.js

```diff
@@ -59,14 +58,46 @@ animalToRemove.remove()
 /************************** EVENTS JS MINI CHALLENGE ******************************/
 
 /***** Deliverable 1 *****/
-function toggleColor(element) {
-    if (element.style.color === "green") {
-        element.style.color = "black"
+
+header.addEventListener("click", function () {
+    if (header.style.color !== 'red') {
+        header.style.color = 'red'
     } else {
-        element.style.color = "green"
+        header.style.color = 'black'
     }
-}
+})
```

Comment:
awesome job on this deliverable! 
depending on the situation, refactoring out the if-else statement into a separate function will help with code readability.